dotnet/src/Connectors/Connectors.MistralAI/Client/MistralFunction.cs

```diff
@@ -140,7 +140,7 @@ private static KernelJsonSchema GetDefaultSchemaForTypelessParameter(string? des
         // If there's a description, incorporate it.
         if (!string.IsNullOrWhiteSpace(description))
         {
-            return KernelJsonSchemaBuilder.Build(null, typeof(string), description);
+            return KernelJsonSchemaBuilder.Build(typeof(string), description);
```

Comment:
Obviously this test won't work in AOT, but is there a subset of the test suite that is run using AOT so that we can verify everything works as expected?

Reply:
Perhaps having a simple acceptance test suite that runs a basic E2E AOT application is running

Reply:
> Obviously this test won't work in AOT, but is there a subset of the test suite that is run using AOT so that we can verify everything works as expected?

This project and many others are not AOT compatible yet. We will add AOT compatibility to one project at a time, starting with a defined subset of projects that we will concentrate on first, and then add the support to the rest of them.

> Perhaps having a simple acceptance test suite that runs a basic E2E AOT application is running.

I plan to have a set of unit tests that will test all new AOT APIs and a test application to make sure all warnings are handled/found.